### ISSUE TITLE
Configuration of SimCLR for Camelyon16

### ISSUE THREAD

**Issue by hrzhang1123** [binli123/dsmil-wsi#15]
Dear Bin Li,
Very interesting work!
Could you please also upload the configuration of SimCLR for training Camelyon16, i.e., epoch, optimizer, lr, weight decay, batch size, etc.
Thank you very much.
BW

**Comment by hrzhang1123** [comment]
Thank you. What's the initial learning rate, please ? Seems the one in the code is quite small that make it converge very slow.
BW
